packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [27.2.0]
+
+### Added
+
+- `CodefiTokenPricesServiceV2` exports `SUPPORTED_CHAIN_IDS`, an array of chain IDs supported by Codefi Price API V2. ([#4079](https://github.com/MetaMask/core/pull/4079))
```

Comment:
Hmm... the whole package exports this constant, not just `CodefiTokenPricesServiceV2`.

I'm curious whether we want to rename this constant to make it more clear that this belongs to the price API? Perhaps `PRICE_API_SUPPORTED_CHAIN_IDS`. Or, perhaps it's worth adding this as a static property to the class, so you would say `CodefiTokenPricesServiceV2.supportedChainIds`. What are your thoughts?

Reply:
Emmm thats right we export `SUPPORTED_CHAIN_IDS` in `CodefiTokenPricesServiceV2` and we export it in `src/index.ts`.
The point of this change is to fix entension imports of SUPPORTED_CHAIN_IDS in extension. I wanted to upgrade assets-controllers on extension to lates and this import
import { SUPPORTED_CHAIN_IDS } from '@metamask/assets-controllers/dist/token-prices-service/codefi-v2';
was not working anymore.
With this change i can import it like this instead
import { SUPPORTED_CHAIN_IDS } from '@metamask/assets-controllers';

Both approaches seem fine to me, i dnt have a preference 🙏

Reply:
I like the middle ground suggestion of updating the Jsdoc to reference to exp:
/**
 * The list of chain IDs that can be supplied in the URL for the `/spot-prices`
 * endpoint, but in hexadecimal form (for consistency with how we represent
 * chain IDs in other places).
 * Used by `CodefiTokenPricesServiceV2`  to validate if the chainId is supported by price-api.
 */

Reply:
I don't believe it would be a breaking change as this export was only internal and wasn't public before, but if we're concerned about it, sure, we can just add the JSDoc comment for now and rename the constant later. I'm sure we'll have an opportunity in the future to do that.

Reply:
Since `SUPPORTED_CHAIN_IDS` was already being used as a `dist/` import, a rename would require changing code downstream in addition to updating import statements. 

I guess technically needing to update import paths alone makes this a breaking change in practice. (Although it probably shouldn't be marked as one in the changelog?) Either way, it seems better to reduce disruption for now. 

We could definitely do the rename down the road, maybe along with some of the other variables and types in the file.

Reply:
Thank you guys 🙏  i ve made the jsdoc suggestion update on this PR https://github.com/MetaMask/core/pull/4083 🙏